charts/federation-v2/README.md

```diff
@@ -83,13 +83,13 @@ chart and their default values.
 
 | Parameter | Description | Default |
 |-----------|-------------|---------|
+| controllermanager.enabled | Specifies whether to enable the controller manager in federation v2 | true |
```

Comment:
@marun 

A use-case for not installing the controller-manager might be that you like the federation APIs but want to write your own controller, for example if you were prototyping a new propagation mechanism. I don't think it's harmful to allow this in the chart, since doing so does not drastically effect the PR and the default is to enable it.

Reply:
@pmorie thanks for the comments, this is exactly a use-case for the helm chart refactor.

@font Yes, I want to put controller manager chart and federation API CRDs into same chart, but due to there maybe same name files, like `service.yaml`, I intend to put them to different charts.

So there are three options here for the helm chart

- Original Topology:

federation-v2
   |
   | -- charts
   |           |
   |           | -- clusterregistry
   |           |
   |           | -- federatedtypes
   |
   | -- controllermanager

- New Topology 1:

federation-v2
   |
   | -- charts
   |           |
   |           | -- clusterregistry
   |           |
   |           | -- controllermanager
   |
   | -- federatedtypes

- New Topology 2:

federation-v2
   |
   | -- charts
   |           |
   |           | -- clusterregistry
   |           |
   |           | -- controllermanager
   |           |
   |           | -- federatedtypes

Any comments for the above proposal?